### DISCUSSION TITLE
Jupyter-like Python cells

### DISCUSSION THREAD

**Discussion by ponyatov** [cristianvasquez/obsidian-lab#4] [Ideas]
Good Day

Is anybody works on adding Jupyter-like cells into Obsidian to use them in cooperation with Python kernel runs in the background?

I think it can be useful for integrating the obsidian JS engine with general-purpose Python runtime (with some i/o gates maybe).

**Comment by cristianvasquez** [comment]
Hello @ponyatov, AFAIK there are some people trying to embed multi purpose Python Runtime: https://forum.obsidian.md/t/jupyter-notebook-integration-for-obsidian/4951/9 . My intention with obsidian-lab was not to do literate programming, but to test new functionality in Obsidian, assisted by ML .

**Comment by cristianvasquez** [comment]
I also wrote  a very simplistic Plugin to run scripts, but it's nothing like jupyter :) https://github.com/cristianvasquez/obsidian-snippets-plugin
